android/src/main/java/io/radar/react/RNRadarUtils.java

```diff
@@ -98,6 +100,26 @@ static int numberForUserInsightsLocationConfidence(RadarUserInsightsLocation.Rad
         return Radar.RadarPlacesProvider.NONE;
     }
 
+    static WritableMap mapForJSONObject(JSONObject obj) {
+        WritableMap map = Arguments.createMap();
+        Iterator<String> keys = obj.keys();
+        while(keys.hasNext()) {
+            String key = keys.next();
+            Object val = obj.get(key);
+            if (val instanceof String) {
+                String valStr = (String)val;
+                map.putString(key, valStr);
+            } else if (val instanceof Number) {
+                Number valNum = (Number)val;
+                map.putInt(key, valNum.intValue());
```

Comment:
Do we only allow `int` values for numbers? `Double` might be better here

Reply:
We do allow doubles, but I think ints are more common. We could use double here since it's more flexible and tell folks to convert to int if more appropriate.